tests/messages/func_w0401_package.txt

```diff
@@ -1,2 +1,2 @@
-R:  1: Cyclic import (input.func_w0401_package.all_the_things -> input.func_w0401_package.thing2)
+R:  1: Cyclic import (tests.input.func_w0401_package.all_the_things -> tests.input.func_w0401_package.thing2)
```

Comment:
This is the same thing as the pyreverse stuff, I think - should we change to the cwd-based fix here too?

Reply:
cwd-based fix in [6fa6e74](https://github.com/PyCQA/pylint/pull/7153/commits/6fa6e74e465fb145c8d80dabd3a1b0987d6d85b3)